src/sentry/integrations/vsts/client.py

```diff
@@ -33,10 +36,11 @@ def __init__(self, identity, oauth_redirect_url, *args, **kwargs):
         if 'access_token' not in self.identity.data:
             raise ValueError('Vsts Identity missing access token')
 
-    def request(self, method, path, data=None, params=None):
+    def request(self, method, path, data=None, params=None, api_preview=False):
         self.check_auth(redirect_url=self.oauth_redirect_url)
+        api_version = self.api_version if api_preview is False else self.api_version + self.api_version_preview
```

Comment:
don't we generally prefer interpolation to concatenation?

Reply:
@macqueen could you explain a bit? I'm not sure what is preferred

Reply:
i just generally thought something like:
```python
 api_version = u'{}{}'.format(self.api_version, self.api_version_preview if api_preview else '')
```
was preferred stylistically